### ISSUE TITLE
Adding accessibility

### ISSUE THREAD

**Issue by fimkap** [lawnstarter/react-native-picker-select#183]
How I'm supposed to add accessibility to the picker? For example, trying it with VoiceOver on, It seems like I have accessibility on select items (after I activate the selection) but I cannot find a way to add it to the selection box itself, so I get the focus on the box but no voice hints.

**Comment by lfkwtz** [comment]
Admittedly I have no idea how accessible or not this component is - my assumption would be not very much unfortunately.

Going to move this to the feature request board. Happy to take PRs or ideas on how to improve it.

**Comment by dquessenberry** [comment]
I realize this ticket is closed and has been added to the Feature Requests, but I wanted to document how accessibility can be added to the RNPickerSelect component currently. IMO, it would be nice to just support accessibility with direct props, which I am happy to implement if there is a desire. After looking through the source code, I noticed that the accessibility attributes can be added via the touchableWrapperProps prop. For example: 

```
import RNPickerSelect from 'react-native-picker-select';

export const Dropdown = () => {
    return (
        <RNPickerSelect
            onValueChange={(value) => console.log(value)}
            items={[
                { label: 'Football', value: 'football' },
                { label: 'Baseball', value: 'baseball' },
                { label: 'Hockey', value: 'hockey' },
            ]}
            touchableWrapperProps={{ // <- Use touchableWrapperProps to pass accessibility properties
                accessible: true,
                accessibilityLabel: 'Favorite sport',
                accessibilityHint: 'Double tap to select an option.',
                accessibilityRole: "combobox",
            }}
        />
    );
};
```

**Comment by izgeorge** [comment]
The picker is missing some accessibility labels on the interactive elements. This includes the close overlay, the chevrons, and if you want something besides "Done" to be announced, that could fall under that as well. Disabled chevrons are also not announced as disabled as the label is missing there. I've created a forked branch with some changes which adds a prop called `modalAccessibilityLabels`

```
modalAccessibilityLabels: shape({
      close: PropTypes.string,
      upArrow: PropTypes.string,
      downArrow: PropTypes.string,
  })
```

Here is a diff of the changes https://github.com/lawnstarter/react-native-picker-select/compare/master...izgeorge:iz/a11y?expand=1

Let me know if this is enough to open a PR. I can add the above mentioned touchable wrapper accessibility props to the docs as well.
